addons/account/models/account_move.py

```diff
@@ -526,6 +530,11 @@ def _sequence_fixed_regex(self):
         states={'draft': [('readonly', False)]},
         help='Defines the smallest coinage of the currency that can be used to pay by cash.',
     )
+    invoice_pdf_report_id = fields.Many2one(
+        comodel_name='ir.attachment',
+        string="PDF report",
+        copy=False,
+    )
```

Comment:
Use a binary field (with `attachment=True` (the default) instead

Reply:
No. We need this for the chatter and also to keep the correct filename

Reply:
And you can access the value directly, you just can't access the attachment behind without the search.

But AFAIK what you need in practice is its value, not the record that is behind it.